config/default.yaml

```diff
@@ -90,6 +90,9 @@ instance:
   customizations:
     javascript: '' # Directly your JavaScript code (without <script> tags). Will be eval at runtime
     css: '' # Directly your CSS code (without <style> tags). Will be injected at runtime
+  robots: |
+    User-agent: *
+    Disallow: /
```

Comment:
Do we really want disallow crawling by default?

Reply:
I figured that since we discover videos and instances without crawlers anyway, that's fine.

Reply:
Maybe add a comment "to allow robots to crawl your instance..."?